src/reversion/management/commands/createinitialrevisions.py

```diff
@@ -1,14 +1,22 @@
-from __future__ import unicode_literals, print_function
+from __future__ import unicode_literals
 
 from optparse import make_option
+try:
+    from collections import OrderedDict
+except ImportError:  # For Python 2.6
+    from django.utils.datastructures import SortedDict as OrderedDict
+
+try:
+    from django.apps import apps
+except ImportError:  # For Django < 1.7
+    from django.db import models as apps
```

Comment:
Yes, but django.db.models includes other things like models.Q, models.Max, which aren’t present in django.apps.

> On 21 May 2015, at 17:14, Claude Paroz <notifications@github.com> wrote:
> 
> In src/reversion/management/commands/createinitialrevisions.py:
> 
> >
> >  from optparse import make_option
> > +try:
> > +    from collections import OrderedDict
> > +except ImportError:  # For Python 2.6
> > +    from django.utils.datastructures import SortedDict as OrderedDict
> > +
> > +try:
> > +    from django.apps import apps
> > +except ImportError:  # For Django < 1.7
> > +    from django.db import models as apps
> 
> I didn't look in details about differences, but it seems at least in Django 1.8 they were equivalent enough to justify some aliasing:
> https://github.com/django/django/blob/stable/1.8.x/django/db/models/__init__.py#L62
> https://github.com/django/django/blob/stable/1.8.x/django/db/models/loading.py
> 
> —
> Reply to this email directly or view it on GitHub.
>